ext/network_interface_ext/netifaces.c

```diff
@@ -200,6 +200,7 @@ static VALUE add_to_family(VALUE result, VALUE family, VALUE value)
   return result;
 }
 
+VALUE rb_cNetworkInterfaceError;
```

Comment:
Wrote this in a separate comment, but just incase it gets lost:

I think you should be able to move both values to line 3 to fix the scope issue/introduce a bit of consistency

```c
#include "ruby.h"
#include "netifaces.h"

VALUE rb_cNetworkInterface
VALUE rb_cNetworkInterfaceError;

#if !defined(WIN32)
#if  !HAVE_GETNAMEINFO
#undef getnameinfo
#undef NI_NUMERICHOST
```